chain/src/main/scala/org/bitcoins/chain/blockchain/ChainHandler.scala

```diff
@@ -367,7 +355,7 @@ case class ChainHandler(
     blockHeaderDAO.getAtHeight(height)
 
   /** @inheritdoc */
-  override def getFilterHeaderCount: Future[Int] = {
+  override def getFilterHeaderCount(): Future[Int] = {
```

Comment:
nit: Since these don't modify the database we shouldn't use `()`. This should be saved for _writes_ to the database.

Reply:
fixed